modules/myclasses/scripts/generate-ssl-certs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ ! -e server.js ]
+then
+	echo "Error: could not find main application server.js file"
+	echo "You should run the generate-ssl-certs.sh script from the main MEAN application root directory"
+	echo "i.e: bash scripts/generate-ssl-certs.sh"
+	exit -1
+fi
+
+echo "Generating self-signed certificates..."
+mkdir -p ./config/sslcerts
+openssl genrsa -out ./config/sslcerts/key.pem 1024
+openssl req -new -key ./config/sslcerts/key.pem -out ./config/sslcerts/csr.pem
+openssl x509 -req -days 9999 -in ./config/sslcerts/csr.pem -signkey ./config/sslcerts/key.pem -out ./config/sslcerts/cert.pem
+rm ./config/sslcerts/csr.pem
```

Comment:
@Bakushin10 why do we need this generate-ssl-certs.sh or was it auto-generated ?